data/arrow.go

```diff
@@ -794,6 +794,9 @@ func (frames Frames) MarshalArrow() ([][]byte, error) {
 	bs := make([][]byte, len(frames))
 	var err error
 	for i, frame := range frames {
+		if frame == nil {
```

Comment:
I think this should return an Error instead since there really shouldn't be nil entries

Reply:
@kylebrandt do you know if it is necessary for this to be an array of frame pointers?